core/geometry/src/polyface/PolyfaceData.ts

```diff
@@ -451,27 +599,31 @@ export class PolyfaceData {
         return false;
     return true;
   }
-  /** Reverse data in entire facet indexing arrays.
-   * * parameterized over type T so non-number data -- e.g. boolean visibility flags -- can be reversed.
+  /**
+   * Reverse data in an indexing array facet-by-facet for all of the facets specified by `facetStartIndex`.
+   * * Parameterized over type T so non-number data (e.g., boolean visibility flags) can be reversed.
+   * @param facetStartIndex start indices of all facets to be reversed.
+   * @param indices the indexing array, e.g., pointIndex, normalIndex, paramIndex, etc.
+   * @param preserveStart `true` to preserve the start of all facets (example: turn facet [1,2,3,4] to [1,4,3,2]);
+   * `false` to reverse all (example: turn facet [1,2,3,4] to [4,3,2,1]).
    */
```

Comment:
reverseIndices and reverseIndicesSingleFacet functions call functions which are located at the end of this file so it's best to move this 2 functions to the end of this file for better code readability. However, I did not move them to make it easier to reveiw. I will move them in the next PR.